### DISCUSSION TITLE
Very slow deleting from Wasabi S3 bucket

### DISCUSSION THREAD

**Discussion by Rich360ctrl** [iterate-ch/cyberduck#15724] [Q&A]
Hi all

I am trying to delete a Veeam archive from Wasabi S3. It's pretty large, about 2.5TB of data.

I can start the delete process, and Cyberduck then speeds through a few deletions, but then it starts to slow down, and gets slower and slower, consuming more and more CPU and RAM. It got up to about 35% CPU and 12GB RAM, and was completing 1 deletion every few minutes.

I realise that it might take a while to burn through that amount of data, but after 2 days of running (and a crash) the bucket size hasn't changed. If each deletion is just one block then I could be waiting an extremely long time...

Any ideas?

Thanks,

Rich.

**Comment by tatze96** [comment]
Please attach the last [crash report](https://docs.cyberduck.io/cyberduck/support/#crash-reports) for further investigation.

Also, if the issue is still present:
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level) and send us the log file after the issue occurs again? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15724 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
